src/containers/LoginPage/index.js

```diff
@@ -1,5 +1,47 @@
-import React from 'react'
+import React, { Component } from 'react'
+import axios from 'axios';
+import View from './View';
 
-export default function LoginPage() {
-  return <p>Loginpage</p>
+export default class LoginPage extends Component {
+  constructor() {
+    super();
+    this.state = {
+      username: '',
+      password: '',
+      isLogin: false
+    };
+  }
+
+  onChangeUsername = (e) => {
```

Comment:
gom `onChangeUsername` và `onChangePassword` thành 1 hàm luôn e nhé :)
```suggestion
  handleChange = (e) => {
    const { name, value } = e.target
    this.setState({ [name]: value })
  }
```